ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncActions.java

```diff
@@ -69,6 +72,13 @@ public FastSyncActions(
     this.ethContext = ethContext;
     this.syncState = syncState;
     this.metricsSystem = metricsSystem;
+    this.trailingPeerLimiter =
+        new TrailingPeerLimiter(
+            ethContext.getEthPeers(),
+            () ->
+                new TrailingPeerRequirements(
+                    syncState.getLocalChainHeight() + syncConfig.getFastSyncPivotDistance(),
```

Comment:
are we sure about that ? If I'm close to the head (<getFastSyncPivotDistance)

Reply:
I changed it to use the min of local + pivot distance or the current best estimate, will do some testing today